### PR TITLE
bpo-34901: add isolated (-I) flag to sys.flags

### PR DESCRIPTION
**Pull request by danishprakash**
https://bugs.python.org/issue34901

### CODE REVIEW

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -345,6 +345,7 @@ always available.
    :const:`debug`                :option:`-d`
    :const:`inspect`              :option:`-i`
    :const:`interactive`          :option:`-i`
+   :const:`isolated`             :option:`-I`
```
Comment (ned-deily) [inline Doc/library/sys.rst:348]: It would also be a good idea to add a `.. versionadded:: 3.4` note down below this.

Comment (danishprakash) [reply Doc/library/sys.rst:348]: thanks for pointing this out, fixed.

**Review by ned-deily — APPROVED** [review]
Thanks for the PR and the update!

**Comment by ned-deily** [comment]
@danishprakash, if you have time and are so inclined, it would be nice if you could do a manual backport to 3.6.

**Comment by danishprakash** [comment]
@ned-deily definitely, I assume it requires me to cherry-pick the commit as mentioned above and then open a PR against the 3.6 branch, is that it?

**Comment by tirkarthi** [comment]
@danishprakash  Yes, you can install cherry_picker from PyPI and run the command mentioned which will help you.

**Comment by ned-deily** [comment]
@danishprakash Yes, that's right.  You can use follow the process [here](https://devguide.python.org/gitbootcamp/#backporting-merged-changes) or just manually use the `git cherry-pick `command.

**Pull request merged**